tests/home.utils.ts

```diff
@@ -0,0 +1,36 @@
+import { Page } from "@playwright/test";
+import * as dotenv from "dotenv";
+
+import { ensureTextualElementExists, wait } from "./utils";
+
+dotenv.config({ path: ".env.testing" });
+
+export async function clickTokenIfAvailable(page: Page) {
+  const token_locator = page.locator("//div/h2[contains(text(),'Tokens')]/..//span/p");
```

Comment:
Is using xPath reliable? If DOM structure changes we'll have to update this.
Better to use selector based on text `Tokens` only.

Reply:
replaced xpath with text based selector